sql/mob_groups.sql

```diff
@@ -8017,9 +8017,9 @@ INSERT INTO `mob_groups` VALUES (22,995,112,'Demon_Warlock',330,0,624,0,0,50,52,
 INSERT INTO `mob_groups` VALUES (23,994,112,'Demons_Elemental',0,128,0,0,0,43,45,0);
 INSERT INTO `mob_groups` VALUES (24,3576,112,'Shadow_Eye',0,32,2226,6700,0,48,49,0);
 INSERT INTO `mob_groups` VALUES (25,693,112,'Chaos_Elemental',0,128,452,3200,0,42,42,0);
-INSERT INTO `mob_groups` VALUES (26,510,112,'Boreal_Hound',300,0,0,28675,0,58,58,0);
-INSERT INTO `mob_groups` VALUES (27,509,112,'Boreal_Coeurl',300,0,0,6750,0,58,58,0);
-INSERT INTO `mob_groups` VALUES (28,511,112,'Boreal_Tiger',300,0,0,15400,0,58,58,0);
+INSERT INTO `mob_groups` VALUES (26,510,112,'Boreal_Hound',15,0,0,28675,0,58,58,0);
```

Comment:
This is incorrect for ASB's era.

https://ffxiclopedia.fandom.com/wiki/Boreal_Coeurl?oldid=1002550